less/navbar.less

```diff
@@ -1191,6 +1191,14 @@
   }
 }
 
+.nav-toolbar--compact {
+  padding-top: (@spacer * 2);
```

Comment:
Is this line necessary? the .nav-toolbar has already that padding.

Reply:
you are right, not needed, will update!